app/code/Magento/Sales/Model/Order/Email/Sender.php

```diff
@@ -44,25 +43,36 @@ abstract class Sender
      */
     protected $addressRenderer;
 
+    /**
+     * App emulation model
+     *
+     * @var \Magento\Store\Model\App\Emulation
+     */
+    protected $_appEmulation;
```

Comment:
Please make this variable private and remove underscore prefix
```suggestion
    private $appEmulation;
```